src/ngraph/runtime/cpu/mkldnn_utils.cpp

```diff
@@ -706,7 +706,22 @@ bool runtime::cpu::mkldnn_utils::can_use_mkldnn_batchnorm_fprop(const ngraph::No
         return false;
     }
 }
+bool runtime::cpu::mkldnn_utils::can_use_conv_auto()
```

Comment:
split this into two methods. get_mkldnn_version() and can_use_conv_auto()

Reply:
sure Will do that.